packages/backend/prisma/schema.prisma

```diff
@@ -7,18 +7,35 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model Post {
-  id        Int     @id @default(autoincrement())
-  title     String
-  content   String?
-  published Boolean @default(false)
-  author    User?   @relation(fields: [authorId], references: [id])
-  authorId  Int?
+model Language {
+  id       Int    @id @default(autoincrement())
+  language String
+  users    User[]
+}
+
+model Option {
+  id        Int        @id @default(autoincrement())
+  option    String
+  users     User[]
+  helpTypes HelpType[]
+}
+
+model HelpType {
+  id       Int    @id @default(autoincrement())
+  helpType String
+  option   Option @relation(fields: [optionId], references: [id])
+  optionId Int
+  users    User[]
 }
 
 model User {
-  id    Int     @id @default(autoincrement())
-  email String  @unique
-  name  String?
-  posts Post[]
+  id         Int        @id @default(autoincrement())
+  userId     Int        @unique
+  chatId     Int 
+  language   Language   @relation(fields: [languageId], references: [id])
```

Comment:
some people speak multiple languages, I was thinking to have 2 different languages: language of the bot, and then when you are offering help you can choose which languages you speak. Maybe this one we can keep as UI language?